pom.xml

```diff
@@ -0,0 +1,621 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Closure Templates</name>
+  <groupId>com.google.template</groupId>
+  <artifactId>soy</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <description>
+    Closure Templates are a client-side and server-side templating system.
+  </description>
+  <url>https://github.com/google/closure-templates</url>
+  <inceptionYear>2009</inceptionYear>
+
+  <parent>
+    <groupId>org.sonatype.oss</groupId>
+    <artifactId>oss-parent</artifactId>
+    <version>9</version>
+  </parent>
+
+  <organization>
+    <name>Google</name>
+    <url>http://www.google.com</url>
+  </organization>
+
+  <developers>
+    <developer>
+      <name></name>
+      <email></email>
+      <organization></organization>
+      <organizationUrl></organizationUrl>
+    </developer>
+  </developers>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/google/closure-templates/issues</url>
+  </issueManagement>
+
+  <mailingLists>
+    <mailingList>
+      <name>closure-templates-discuss</name>
+      <subscribe>closure-templates-discuss+subscribe@googlegroups.com</subscribe>
+      <unsubscribe>closure-templates-discuss+unsubscribe@googlegroups.com</unsubscribe>
+      <post>closure-templates-discuss@googlegroups.com</post>
+      <archive>https://groups.google.com/forum/#!forum/closure-templates-discuss</archive>
```

Comment:
Better use the "permalink" IMO: `https://groups.google.com/d/forum/closure-templates-discuss`